app/admin/RTMediaAdmin.php

```diff
@@ -1229,13 +1229,14 @@ public static function get_current_tab() {
 		 * @return void
 		 */
 		public function render_page( $page_name, $option_group = null ) {
-			?>
 
+			 $align = is_rtl() ? 'alignleft' : 'alignright';
+			?>
 			<div class="wrap bp-media-admin <?php echo esc_attr( $this->get_current_tab() ); ?>">
 				<div id="icon-buddypress-media" class="icon32"><br></div>
 				<div>
 					<h2 class="nav-tab-wrapper"><?php $this->rtmedia_tabs(); ?>
-						<span class="alignright by">
+						<span class="<?php printf( esc_html( $align ) ); ?> by">
```

Comment:
Here need to use `esc_attr` not `esc_html` like this
We can also write this way `<?php echo esc_attr( $align ); ?>` instead of `<?php printf( esc_html( $align ) ); ?>`

Reply:
On it 👍